### DISCUSSION TITLE
What dose the table's number mean? In Chapter 7.1

### DISCUSSION THREAD

**Discussion by kstcmcjw** [nrc-cnrc/EGSnrc#856] [Questions] [answered]
What dose the table's number mean? In Chapter 7.1

Hello everyone, i begin to study the Chapter 7 BEAMnrc.

In this Chapter 7.1 Build a new BEAMnrc accelerator, in page 55 the table as show:

![image](https://user-images.githubusercontent.com/96934217/158922071-3d7a2fd3-1172-458e-81da-c95eb00690f0.png)

in this table, i know the 40,50 and 51,61 stand for the upper y-jaws and the x-jaws separately,

and 40,50,51,61 is the distance to the source.

But what are the -2, -1, -0.05, 0.05, 1, 2 stand for ? It is in the red rectangle. Whether they are stand for the surface area of the exposure field in different situation?

**Comment by rtownson** [comment, marked as answer]
This is chapter 7 in the getting started manual, and it describes the SYNCJAWS field positions. See section 15.3.8 of the BEAMnrc user's manual, where it explains each of the inputs in detail. Essentially the 4 numbers per line are the (1) "front" (or top) x or y coordinate (depending on jaw orientation) of the positive jaw (2) "back" (or bottom) x or y coordinate (depending on jaw orientation) of the positive jaw, and then the same front/back pair for the negative jaw. If the front and back coordinates are the same, then the jaw face is vertical. 

If you're unsure, a good way to explore this would be to score a phase-space below your jaws, and then take a look at the scatter plot of particle positions using beamdp_gui.

**Reply by kstcmcjw** [reply]
Thank you for answer, by the way, i would know whether the EGSnrc has the professional manual? Getting start PDF  is a fast manual.
Such as BEAM and DOSXYZ has it's detailed user manual for learner.

**Reply by rtownson** [reply]
You can find all the documentation here: https://nrc-cnrc.github.io/EGSnrc/
